### PR TITLE
Final touches before presentation

### PR DESCRIPTION
**Pull request by pataruco**
On this PR:
- Update date and location of the presentation
- improve UX and a11y 
- Move navigation to the footer
- Add external links
- Re-enabled pre-commit hooks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,11 +62,12 @@
   "name": "rusty-azure",
   "private": true,
   "scripts": {
-    "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "postbuild": "react-snap"
+    "postbuild": "react-snap",
```
Comment (barwick) [inline package.json:67]: This looks handy

File: src/components/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+`;
+
+const Navigation: React.FC = () => {
+  const [isHome, setIsHome] = useState(true);
+  const {
+    location: { pathname },
+  } = useHistory();
+
+  useEffect(() => {
+    pathname === '/about' ? setIsHome(false) : setIsHome(true);
```
Comment (barwick) [inline src/components/footer.tsx]:
```suggestion
    setIsHome(pathname !== '/about');
```

or maybe `pathname === '/'`

**Review by barwick — APPROVED** [review]
Es magnífico. One tiny comment

File: src/components/footer.tsx (hunk 2 above)
Comment (pataruco) [reply src/components/footer.tsx]: Nice one!

File: package.json (hunk 1 above)
Comment (pataruco) [reply package.json:67]: It does!

**Pull request merged**